### PR TITLE
Fix Purchase Example docs

### PR DESCRIPTION
**Pull request by awholegunch**
Fix for https://issues.cask.co/browse/CDAP-1844

Staged at: [Purchase Example Documentation](http://stg-web101.sw.joyent.continuuity.net/cdap/2.8.0-SNAPSHOT-r2.8_fix_purchase_docs/en/examples-manual/examples/purchase.html)

### CODE REVIEW

File: cdap-docs/examples-manual/source/examples/purchase.rst [hunk 1]
```diff
@@ -123,6 +124,16 @@ Running CDAP Applications
 Running the Example
 ===================
 
+In the commands that follow, we'll assume that you will run all commands
+from the example's base directory (``examples/``\ |literal-example|\ ``/``, in the
+Standalone CDAP SDK directory).
+
+For brevity, we will simply use ``cdap-cli.sh`` for the Command Line Interface. Substitute
+the actual path of ``../../bin/cdap-cli.sh``, or ``..\..\bin\cdap-cli.bat`` on Windows, as
+appropriate. Other scripts that are supplied in teh examples will also have Windows
```
Comment (anwar6953) [inline cdap-docs/examples-manual/source/examples/purchase.rst]: You misspelled teh word.

Comment (awholegunch) [inline cdap-docs/examples-manual/source/examples/purchase.rst]: I did do that! thanks...

File: cdap-docs/examples-manual/source/examples/purchase.rst [hunk 2]
```diff
@@ -151,45 +159,41 @@ Once the application is deployed:
 
 - Click on *PurchaseHistory* in the Overview page of the CDAP Console to get to the
   Application detail page, click *PurchaseHistoryService* in the *Service* pane to get to the
-  Service detail page, then click the *Start* button; do the same for the *CatalogLookupService*; or
-- From the Standalone CDAP SDK directory, use the Command Line Interface:
+  Service detail page, then click the *Start* button; do the same for the *CatalogLookupService*
```
Comment (bdmogal) [inline cdap-docs/examples-manual/source/examples/purchase.rst]: Is *PurchaseHistoryService* missing here?

File: cdap-docs/examples-manual/source/examples/purchase.rst [hunk 3]
```diff
@@ -26,28 +26,28 @@ and write to an ObjectStore Dataset.
...
-    *history* Dataset every morning at 4:00 A.M. using a Time Schedule, and also every time 1MB of data
+    ``history`` Dataset every morning at 4:00 A.M. using a Time Schedule, and also every time 1MB of data
     is ingested by the ``purchaseStream`` using a Data Schedule.
   - You can either manually (in the Process screen of the CDAP Console) or 
     programmatically execute the ``PurchaseHistoryBuilder`` MapReduce to store 
-    customers' purchase history in the *history* Dataset.
-  - Request the ``PurchaseHistoryService`` retrieve from the *history* Dataset the purchase history of a user.
-  - Execute a SQL query over the *history* Dataset. You can do this using a series of ``curl``
+    customers' purchase history in the ``history`` Dataset.
+  - Request the ``PurchaseHistoryService`` retrieve from the ``history`` Dataset the purchase history of a user.
```
Comment (bdmogal) [inline cdap-docs/examples-manual/source/examples/purchase.rst]: This sentence seems malformed/has a typo - perhaps a missing "to" before retrieve?

**Comment by bdmogal** [comment]
Minor comment - LGTM otherwise

**Comment by bdmogal** [comment]
:+1:

**Pull request merged**